### ISSUE TITLE
Migrate to single chat

### ISSUE THREAD

**Issue by dcwalk** [edgi-govdata-archiving/overview#46]
Based on discussion we will be going ahead with this, to mark as complete we need to:
- [X] Notify, create/rename new `#dev` channel in Archivers repo
- [x] Notify the participants in the other channels
  - [X] Civic Tech Toronto
  - [x] EDGI
  - [x] DataRefuge
- [x] Update overview documentation
- [x] Add details to EDGI website
- ~Test portal integration with established rooms in other orgs~

- - - 
Right now our development chat is spread out over:
- Civic Tech Toronto slack
- DataRefuge slack
- EDGI
- (now) Archivers slack

Other places we work:
- This github organization

We need to reduce the pain cause by all the chats!

**Comment by titaniumbones** [comment]
+10^3

On February 5, 2017 11:49:04 AM EST, dcwalk <notifications@github.com> wrote:
>Right now our development chat is spread out over:
>- Civic Tech Toronto slack
>- DataRefuge slack
>- EDGI
>- (now) Archivers slack
>
>Other places we work:
>- This github organization
>
>We need to reduce the pain cause by al the chats!
>
>-- 
>You are receiving this because you are subscribed to this thread.
>Reply to this email directly or view it on GitHub:
>https://github.com/edgi-govdata-archiving/overview/issues/46

-- 
Sent from my Android device with K-9 Mail. Please excuse my brevity.

**Comment by dcwalk** [comment]
I think we want for our **technical development** communications:
- stuff open for everyone to join, 
- a _clear indication_ of where tool development conversations will happen
- something most of us are already using would be preferable, and 
- bonus if it integrates well with github?

I think this is a priority before most of the upcoming events.

Do people think we want slack? Do we have a sense on the best of our slacks to use?

Based on the criteria above my thinking on a ranking is:

1. Archivers
1. Data Refuge
1. Civic Tech Toronto
1. EDGI

I like the idea of not taking over someone else space (e.g. the bottom three WERE ALL THAT), and having a quick way for anyone to join (e.g. Archivers and Civic Tech slack have the heroku invite app). But there are also so many people that signed up to all of these for other purposes. It might be odd to re-purpose a slack team?

**Comment by titaniumbones** [comment]
I think we need to be clear that we talking here about **technical development activity**, not about general co-ordination of projects like EDGI and DataRefuge, or even (necessarily) event organization.  I quite liked the NYC model: co-ordinators discussiong on an open channel in an only-moderately-open slack team; but all event activity happening in a completely different, wide-open, no-invite-required team.

So, my vote would be to try to move all dev work to Archivers. This is something we should talk over w/ at least the major city rescue organizing teams, if we can.  I think we could do it pretty easily, at least from the civic Tech side; NYC is already there; lotso f the key devs from philly were at NYC... it seems feasible to me.  

Might need to do some re-namespacing on channels.

**Comment by dcwalk** [comment]
Yes, thank you @titaniumbones for raising that point-- I meant only about technical development communication, I've edited above. 

The other coordination, especially event, feels well handled by current approaches! Just trying to lump all our dev talk in the same place :)

**Comment by sonalranjit** [comment]
Ya i agree with a single chat for all development. I would suggest to migrate to whichever slack has most of the devs, if its really scattered then create a new one?

**Comment by mhucka** [comment]
My $0.02 is that I agree too, and have been having a hard time (= wasting time) trying to catch up periodically by looking in roughly 20 channels spread over 2 slack teams. So whatever we can do to consolidate would help, IMHO.

**Comment by patcon** [comment]
If it's valuable, after choosing the go-to slack team, you can still connect up "channel-to-channel portals" using this tool:
https://github.com/CivicTechTO/slackbridge

It might help keep your door open to folks not invested enough to sign into a specific slack team. But yes, more overhead ;)

_e.g. Civic Tech Toronto's slack has a `#portal-civictechottawa` channel and Civic Tech Ottawa's slack has a channel `#portal-civictechto` through which msgs are passed._

**Comment by b5** [comment]
Wow that slack bridge tool is _very_ appealing to me. I love the idea of bridging EDGI & DataRefuge into archivers so they can be kept abreast of ongoings. Would be a fantastic way to keep communities connected as well. @patcon are there any "technical hiccups" that make this less-than seamless?

**Comment by titaniumbones** [comment]
should we give it a try with some test channels? Maybe the sf next-steps
channel?

**Comment by titaniumbones** [comment]
and @patcon any chance you could help with setting this up?

**Comment by dcwalk** [comment]
Okay, not seeing much more discussion I'll begin the migration to Archivers.

I'd like to rename the current `#next-steps` to `#dev` We can have `#dev-pagefreezer`, `#dev-pipeline` etc... as needed, but maybe `#dev` is where we test a bridge to DataRefuge and EDGI slacks?

Can we do a thumbs up/thumbs down on this? (Using the emoji on the comment?) 

cc @sonalranjit @titaniumbones @mhucka @b5 et al...

**Comment by titaniumbones** [comment]
I see @dcwalk is undecided.

**Comment by dcwalk** [comment]
Haha, no just cueing the vote :)

**Comment by dcwalk** [comment]
(but generally yes)

**Comment by patcon** [comment]
@titaniumbones I'm spread a little thinly, but if I can't do it myself, I can walk someone else through the complicated parts, and update docs accordingly :)

**Comment by titaniumbones** [comment]
I've installed heroku tools & git cloned the civic tech repo & will try, but am also pushing my limits so
if someone who is not @b5 can do it quicker than me go ahead!

Patrick Connolly <notifications@github.com> writes:

> @titaniumbones I'm spread a little thinly, but if I can't do it myself, I can walk someone else through the complicated parts, and update docs accordingly :)
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly, view it on GitHub, or mute the thread.


--

**Comment by dcwalk** [comment]
I can take a pass at it, but in my mind it is not an immediate priority to get the portal up and running. I think the migration is more time sensitive and we should got ahead and do some mopping up next week? 

@titaniumbones and @b5 Don't work on it now! Lets leave till next week, and I've some spare cycles so I'll aim for a first pass

**Comment by titaniumbones** [comment]
I like that @dcwalk, I think you're right.

**Comment by patcon** [comment]
@dcwalk :100: agreed. The portals I've set up have not been super-active. (That's largely because the slack teams on the other side aren't.)

**Comment by ambergman** [comment]
@dcwalk Thanks for taking lead on this - I'm sorry I got tied up with infinity other nonsense things and never really wrote out my reasons for why I prefer going to the EDGI slack, but I think this makes sense for now. I think working to build an identity for all of the different development projects is important, so that everyone feels like they're part of the movement not just a helper to it, but hopefully we'll still be able to get that going on archivers - and, as everyone is noting, there will likely be transitions upcoming (and the bridge tool seems awesome and will definitely be useful). Thanks again!

**Comment by dcwalk** [comment]
Update on this --
- The migration happened, dev convo is fairly regularly happening in "Archivers"
- Waiting on testing portals while confirming details with both EDGI and DataRefuge in terms of possible approaches

**Comment by dcwalk** [comment]
Yoooooo @patcon -- this is that stalled issue, lets get this ball rolling! :)

**Comment by patcon** [comment]
Going to close this as largely done, and create a new ticket for slack bridge :) Feel free to re-open if that feels wrong
